src/pyhf/infer/calculators.py

```diff
@@ -59,6 +59,20 @@ def __init__(self, shift):
         self.shift = shift
         self.sqrtqmuA_v = None
 
+    def cdf(self, value):
+        """
+        Compute the value of the cumulative distribution funcation for a given value of the test statistic.
```

Comment:
```suggestion
        Compute the value of the cumulative distribution function for a given value of the test statistic.
```

Reply:
I think that might have been my bad. Sorry.